Actions/IncrementVersionNumber/IncrementVersionNumber.ps1

```diff
@@ -30,20 +30,18 @@ try {
 
     $settings = $env:Settings | ConvertFrom-Json
 
-    # Ensure the repoVersion setting exists in the repository settings. Defaults to 1.0 if it doesn't exist.
-    Set-VersionInSettingsFile -settingsFilePath (Join-Path $baseFolder $RepoSettingsFile) -settingName 'repoVersion' -newValue $settings.repoVersion -Force # $RepoSettingsFile is defined in AL-Go-Helper.ps1
-
-    # Change repoVersion in repository settings
-    Set-VersionInSettingsFile -settingsFilePath (Join-Path $baseFolder $RepoSettingsFile) -settingName 'repoVersion' -newValue $versionNumber
-
     $projectList = @(GetProjectsFromRepository -baseFolder $baseFolder -projectsFromSettings $settings.projects -selectProjects $projects)
 
     $allAppFolders = @()
     foreach($project in $projectList) {
         $projectPath = Join-Path $baseFolder $project
 
-        # Set repoVersion in project settings (if it exists)
         $projectSettingsPath = Join-Path $projectPath $ALGoSettingsFile # $ALGoSettingsFile is defined in AL-Go-Helper.ps1
+
+        # Ensure the repoVersion setting exists in the project settings. Defaults to 1.0 if it doesn't exist.
+        Set-VersionInSettingsFile -settingsFilePath $projectSettingsPath -settingName 'repoVersion' -newValue $settings.repoVersion -Force
```

Comment:
Why force add repoVersion to projects? 🤔 repoVersion could also live in the repo settings (like in BCApps)

Reply:
(see the other comment)

Yes, yes, the action can check if it's changing the version of _all_ projects and `repoVersion` is defined in repo settings and only change that. But then it still has to go through all the projects settings and change it there as well (in case it is defined on project level as well).

Reply:
Perhaps.. But wouldn't it also mean that if we run this in BCApps we'd add the repoVersion into all projects? In the case of BCApps we would like the repoVersion to be the same in all projects. Doesn't it then make sense to have repoVersion on repo level rather than on project level?

Reply:
Yes, it does, but it is more of a cosmetic issue.